stock_warehouse_analytic_purchase/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+Adds to the analytical fields and analytical tags of the lines that are automatically calculated
```

Comment:
Please rewrite description and make it more detailed (this definition doesn't refer to warehouses, and does not indicate that this new definition has higher priority and default analytic rules)